samples/VanillaJSTestApp2.0/app/onPageLoad/auth.js

```diff
@@ -0,0 +1,103 @@
+// Browser check variables
+// If you support IE, our recommendation is that you sign-in using Redirect APIs
+// If you as a developer are testing using Edge InPrivate mode, please add "isEdge" to the if check
+const ua = window.navigator.userAgent;
+const msie = ua.indexOf("MSIE ");
+const msie11 = ua.indexOf("Trident/");
+const msedge = ua.indexOf("Edge/");
+const isIE = msie > 0 || msie11 > 0;
+const isEdge = msedge > 0;
+
+let signInType;
+
+// Create the main myMSALObj instance
+// configuration parameters are located at authConfig.js
+const myMSALObj = new msal.PublicClientApplication(msalConfig); 
+
+// Register Callbacks for Redirect flow
+myMSALObj.onRedirectAppLoad().then((tokenResponse) => {
+    const accountObj = tokenResponse ? tokenResponse.account : myMSALObj.getAccount();
+    if (accountObj) {
+        console.log('id_token acquired at: ' + new Date().toString());
+        showWelcomeMessage(accountObj);
+        seeProfileRedirect();
+    } else if (tokenResponse && tokenResponse.tokenType === "Bearer") {
+        console.log('access_token acquired at: ' + new Date().toString());
+    } else {
```

Comment:
Can you add comments documenting what each of these scenarios indicate?

Reply:
done.